modules/filter/texinfo.cpp

```diff
@@ -26,7 +26,7 @@ namespace {
     struct Table {
       String name;
       bool ignore_item;
-      Table(String n) : name(n), ignore_item(false) {}
+      Table(const String &n) : name(n), ignore_item(false) {}
```

Comment:
I use `ParmStr` (_common/parm_string.hpp_) when passing in strings to avoid an unnecessary copy when a c-string is passed in.  This should be `Table(ParmStr n)`.